.github/scripts/trymerge.py

```diff
@@ -364,6 +371,7 @@
     re.MULTILINE
 )
 RE_DIFF_REV = re.compile(r'^Differential Revision:.+?(D[0-9]+)', re.MULTILINE)
+CIFLOW_LABEL = re.compile(r"ciflow/*")
```

Comment:
Minor: This regex looks a bit off:

* If you just want it to match a string starting with `ciflow/`, r"^ciflow/" will do
* If you are looking to match the whole string like `ciflow/periodic`, the regex should be `ciflow/.*` or `ciflow/.+` or ciflow/\w*`

Reply:
Huh, I'm surprised that the regex works the way it is right now (and it does work, I double checked).

Turns out that re.match() only matches from the start of the string.  Still, would be good to make the regex more explicit.